.github/workflows/r_linux.yml

```diff
@@ -0,0 +1,109 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Linux R/C++ bundled static build
+
+on:
```

Comment:
Because the number I would not run all of these builds on each PR commit, although its feedback seems pretty useful, so I'd pick a single build (a relevant one, perhaps `ubuntu-gcc-release`) and configure it in `workflows/r.yml`. This way we'd have a build running on each PR commit possibly catching valid issues.

The rest of the builds should run on either `push to master` or as a cron schedule. @kou opinion?
If want to the remaining builds as cron jobs, then we should place them to `r_cron.yml`, otherwise they should go to `r.yml` using the `if: github.event_name == 'push'` condition for the job.

Reply:
I can replace the `ubuntu-r` job in `r.yml` with one of these, sounds good.

IMO push to master sounds good, until we start seeing GHA backups and need to throttle it more. The trouble with running on master/cron is: how do we get alerted about failures? IIUC we aren't checking the existing cron GHA jobs for failures to include in our daily crossbow email. 

If we're not running these on PR commit, then they should be wired in crossbow for on-demand use in a PR review, right? Do we have similar setup for the existing GHA cron jobs?

Reply:
Anyone subscribed to the apache/arrow repository's notifications should receive build error notifications for the cron jobs. For example I'm getting a couple ones per day for my fork.

That wiring is manual, we have crossbow jobs defined for most of the docker-compose builds regardless whether we run them as GHA jobs or not. Which means that we can trigger docker builds with comments, even the ones we regularly run on GHA.

To summarize, you're free to choose where to run these docker builds, either via crossbow nightlies for better notifications, of via GHA actions.